mlflow/tracking/fluent.py

```diff
@@ -1767,14 +1769,14 @@ def _get_or_start_run():
 
 
 def _get_experiment_id_from_env():
-    experiment_name = env.get_env(_EXPERIMENT_NAME_ENV_VAR)
-    experiment_id = env.get_env(_EXPERIMENT_ID_ENV_VAR)
```

Comment:
Is `env` used anywhere else? If not, let's remove it.

Reply:
looks like it's not being used anywhere else, and I forgot to remove the import :)